test/deploy-kubeflow.sh

```diff
@@ -56,4 +56,11 @@ ${KUBEFLOW_SRC}/scripts/kfctl.sh apply platform
 ${KUBEFLOW_SRC}/scripts/kfctl.sh generate k8s
 ${KUBEFLOW_SRC}/scripts/kfctl.sh apply k8s
 
+pushd ks_app
```

Comment:
should we instead update the version in kfctl instead?
the change here will make the test environment different from production environment so test pass here might not mean it's working.

Reply:
Done https://github.com/kubeflow/kubeflow/pull/3335

Reply:
thanks. I think this PR is ready to go once this file is reverted.